Rakefile

```diff
@@ -4,5 +4,20 @@ require 'middleman-gh-pages'
 require 'rspec/core/rake_task'
 RSpec::Core::RakeTask.new(:spec)
 
+desc "Compile javascript -> coffeescript"
+task :coffee do
+  destination = "source/javascripts/refills/coffeescript"
+  Dir["source/javascripts/refills/**/*.js"].each do |source|
+    filename = File.basename(source.gsub(%r{.js}, ".coffee"))
```

Comment:
Since this regex does not include `/` characters, should we use `/` as the delimiter?